stacks-signer/src/signerdb.rs

```diff
@@ -597,7 +654,7 @@ mod tests {
         let db_path = tmp_db_path();
         let db = SignerDb::new(db_path).expect("Failed to create signer db");
         assert_eq!(
-            query_row(&db.db, "SELECT sqlite_version()", NO_PARAMS).unwrap(),
+            query_row(&db.db, "SELECT sqlite_version()", []).unwrap(),
```

Comment:
I created an equivalent const to keep the naming the same as before, because it got removed on the rusqlite version I updated to. I'll let it as reference if you think we should have that everywhere, https://github.com/stacks-network/stacks-core/blob/0a73ad090c01ece6d12d4eb042b0ef664cdaaa9f/stacks-common/src/types/sqlite.rs#L28

Besides this, LGTM

Reply:
I think adding that constant was a useful way to lower the lines of code changed in your PR, but over time, I think we should just use`[]` instead.